enochecker3/enochecker.py

```diff
@@ -388,6 +390,8 @@ def decorator(f: Callable[..., Any]) -> None:
 
             self._dependency_injections[key] = f
 
+            return f
```

Comment:
just need to return the wrapped function so the next decorator can pick it up again